docs/features/techdocs/using-cloud-storage.md

```diff
@@ -30,20 +30,34 @@ techdocs:
     type: 'googleGcs'
 ```
 
-**2. GCP (Google Cloud Platform) Project**
+**2. GCS Bucket**
 
-Create or choose a dedicated GCP project. Set
-`techdocs.publisher.googleGcs.projectId` to the project ID.
+Create a dedicated Google Cloud Storage bucket for TechDocs sites.
+techdocs-backend will publish documentation to this bucket. TechDocs will fetch
+files from here to serve documentation in Backstage. Note that the bucket names
+are globally unique.
+
+Set the name of the bucket to `techdocs.publisher.googleGcs.bucketName`.
```

Comment:
This text seems to imply that the name of the bucket should be this string. You probably meant something like ```Set the app-config value `techdocs.publisher.googleGcs.bucketName` to the name of the bucket you created.```

Reply:
Good catch.